packages/lesswrong/integrationTests/integrationTestSetup.ts

```diff
@@ -56,7 +56,7 @@ async function oneTimeSetup() {
   initGraphQL();
 }
 
-jest.setTimeout(20000);
+jest.setTimeout(50000);
```

Comment:
This is needed as the tests now run slower (probably 50000 is overkill). Hopefully we can lower this again in after some fixes in the future.